### ISSUE TITLE
problem with steam_guard.json

### ISSUE THREAD

**Issue by djskeletik** [bukson/steampy#247]
I've done all instructions which you gave, but today my bot stopped and he isnt working since yesterday, there is a problem with json decoder. What should i do with my code?

Exception:
Traceback (most recent call last):
  File "C:\Users\***\PycharmProjects\****\main.py", line 24, in <module>
    Steam = Steam_client_bot()
            ^^^^^^^^^^^^^^^^^^
  File "C:\Users\***\PycharmProjects\****\steam_con.py", line 20, in __init__
    self.initSteam()
  File "C:\Users\***\PycharmProjects\****\steam_con.py", line 32, in initSteam
    steam_client.login(self.steam_cfg[1], self.steam_cfg[2], config.path + 'steam_guard.json')
  File "C:\Users\***\PycharmProjects\****\venv\Lib\site-packages\steampy\client.py", line 46, in login
    LoginExecutor(username, password, self.steam_guard['shared_secret'], self._session).login()
  File "C:\Users\***\PycharmProjects\****\venv\Lib\site-packages\steampy\login.py", line 20, in login
    login_response = self._send_login_request()
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\***\PycharmProjects\****\venv\Lib\site-packages\steampy\login.py", line 29, in _send_login_request
    rsa_params = self._fetch_rsa_params()
                 ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\***\PycharmProjects\****\venv\Lib\site-packages\steampy\login.py", line 53, in _fetch_rsa_params
    data={'username': self.username}).json()
                                      ^^^^^^
  File "C:\Users\***\PycharmProjects\****\venv\Lib\site-packages\requests\models.py", line 975, in json
    raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)
requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

**Comment by djskeletik** [comment]
The problem looks to be here:         

https://github.com/bukson/steampy/blob/master/steampy/login.py#L51:
```
key_response = self.session.post(SteamUrl.STORE_URL + '/login/getrsakey/',
data={'username': self.username}).json()
```

Now it receives HTML with webstore  content rather than JSON file. Any ideas how to fix it?

**Comment by woctezuma** [comment]
- #241

**Comment by bukson** [comment]
Closed with fix in 0.80
